pkg/bindings/images/types.go

```diff
@@ -182,6 +182,10 @@ type PullOptions struct {
 	Policy *string
 	// Password for authenticating against the registry.
 	Password *string
+	// ProgressWriter is a writer where push progress are sent.
+	// Since API handler for image push is quiet by default, WithQuiet(false) is necessary for
```

Comment:
It seems to be typo.

```suggestion
	// ProgressWriter is a writer where pull progress are sent.
	// Since API handler for image pull is quiet by default, WithQuiet(false) is necessary for
```

Also Image pull handler may be not  quiet by default.

https://github.com/containers/podman/blob/main/pkg/api/handlers/libpod/images_pull.go#L40-L41